worlds/sc2/docs/fr_Starcraft 2.md

```diff
@@ -0,0 +1,87 @@
+# *Starcraft 2*
+
+## Quel est l'effet de la randomisation sur ce jeu ?
+
+Les éléments qui suivent sont les *items* qui sont randomisés et qui doivent être débloqués pour être utilisé dans le 
+jeu:
+1. La capacité de produire des unités, outre les drones/probes/scv.
+2. Des améliorations spécifiques à certaines unités incluant quelques combinaisons qui ne sont pas disponibles dans les 
+campagnes génériques, comme le fait d'avoir les deux types d'évolution en même temps pour une unité *Zerg* et toutes 
+les améliorations de la Lance d'Adun simultanément pour les *Protoss*.
```

Comment:
Spear of Adun est un nom propre selon moi

Reply:
True. Also changed Hyperion for that same reason. Kept "Léviathan", except changing "le" for "un", since it is one among many, in theory. Is that acceptable?

Reply:
Fine by me, especially since Leviathan's translation is pretty much the exact same word. No one will get confused. Spear on the other hand would be more annoying I think